nilearn/maskers/nifti_labels_masker.py

```diff
@@ -705,19 +708,55 @@ def transform_single_imgs(self, imgs, confounds=None, sample_mask=None):
             # ids does not include background label
             region_ids[i] = ids[i]
 
+        self.region_names_ = None
         if self.labels is not None:
+            lower_case_labels = {x.lower() for x in self.labels}
+            knwon_backgrounds = {"background"}
```

Comment:
This could be expanded if we encounter atlases that use another "keyword" for the background (bckgrd, bg...)

Reply:
discussed with @mtorabi59:
- to make our life easier we should probably require that list of labels passed to the constructors MUST include `background` or `Background` and that this should be the first item in the list. This cannot be done in this PR and should be part of some refactoring to standardize our atlases with deprecation cycle: won't break the API but will change some of the objects returned by the fetchers.